src/lib.rs

```diff
@@ -123,10 +123,13 @@ impl Drop for PDB {
 }
 
 impl PDB {
-    pub fn new(is_64bit: bool) -> Result<Self, Error> {
+    pub fn new2(is_64bit: bool, age:u32, signature:u32, mut guid_sig:[u8; 16]) -> Result<Self, Error> {
```

Comment:
Depending on whether those parameters are optional or not this should be one function or modified with a feature.

Reply:
ah I wasnt sure how to expose it since my intent was a single overloaded constructor.